frontend/app/framework/angular/http/http-extensions.ts

```diff
@@ -80,8 +80,10 @@ export module HTTP {
 
 export const pretifyError = (message: string) => <T>(source: Observable<T>) =>
     source.pipe(catchError((response: HttpErrorResponse) => {
-        const error = parseError(response, message);
+        const localizer = LocalizerService.getInstance();
```

Comment:
See other comments below.

Reply:
I dont see how we can do this with dependency injection, unless we edit the 157 uses of the pretityError.

Maybe you can help me out with this one?

Reply:
Commented in the forum.